itest/integration_test.go

```diff
@@ -118,4 +118,14 @@ func testGetInfo(t *harnessTest) {
 	// Ensure network field is set correctly.
 	expectedNetwork := t.tapd.cfg.NetParams.Name
 	require.Equal(t.t, expectedNetwork, resp.Network)
+
+	// Attempt to get the info using the CLI.
+	respGeneric, err := ExecTapCLI(ctxt, t.tapd, "getinfo")
```

Comment:
👍 could also (as part of the PoC) assert a path where the command fails and returns an err, non-blocking

would also be nice to add some basic coverage on all commands in a future PR